cogs/commands/xkcd.py

```diff
@@ -0,0 +1,89 @@
+import json
+import logging
+import random
+from io import BytesIO
+
+import aiohttp
+import discord
+from discord.ext import commands
+from discord.ext.commands import Bot, Context
+
+from config import CONFIG
+
+LONG_HELP_TEXT = """
+For all your xkcd needs
+
+Use /xkcd <comicID> to gets the image of a comic with a specific ID.
+Or just use /xkcd to get a random comic.
+If an invalid arguement is made a random comic is returned
+"""
+
+SHORT_HELP_TEXT = "For all your xkcd needs"
+
+
+class XKCD(commands.Cog):
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.hybrid_command(help=LONG_HELP_TEXT, brief=SHORT_HELP_TEXT)
+    async def xkcd(self, ctx: Context, comic_id: int = -1):
```

Comment:
The argument here should probably be `comid_id: int | None = None`. Using sentinal values is bad.